api-audit/src/main/java/com/capitalone/dashboard/evaluator/RegressionTestResultEvaluator.java

```diff
@@ -51,226 +75,166 @@ public RegressionTestResultEvaluator(TestResultRepository testResultRepository,
 
     @Override
     public Collection<TestResultsAuditResponse> evaluate(Dashboard dashboard, long beginDate, long endDate, Map<?, ?> dummy) throws AuditException {
-
-        List<CollectorItem> testItems = getCollectorItems(dashboard, "codeanalysis", CollectorType.Test);
+        this.beginDate = beginDate-1;
+        this.endDate = endDate+1;
+        this.dashboard = getDashboard(dashboard.getTitle(), DashboardType.Team);
+        this.featureWidget = getFeatureWidget();
```

Comment:
You shouldn't be required to have a feature widget to use this functionality. If I have a dashboard that doesn't have a feature widget this throws and exception

Reply:
@Sbrenthughes feature widget configuration is needed if team wants to know traceability details. Moved this assignment to traceability method and exception is handled as well.